docs/source/examples/DogsVsCats.rst

```diff
@@ -109,6 +109,16 @@ to TensorBay for sharing, reuse, etc.
    dataset_client = gas.upload_dataset(dataset, jobs=8, skip_uploaded_files=False)
    dataset_client.commit("Dogs vs. Cats")
 
+Remember to execute the :ref:`features/version_control:Commit` step after uploading.
```

Comment:
The double quotes in the commit message is not well paired.

Reply:
Request changes

Reply:
There are two `in` in the commit message,
Maybe change the second one to `of` ?